qsiprep/utils/shm.py

```diff
@@ -934,7 +934,7 @@ def anisotropic_power(sh_coeffs, norm_factor=0.00001, power=2, non_negative=True
     Calculate AP image based on a IxJxKxC SH coefficient matrix based on the
     equation:
     .. math::
-        AP = \sum_{l=2,4,6,...}{\frac{1}{2l+1} \sum_{m=-l}^l{|a_{l,m}|^n}}
+        AP = \\sum_{l=2,4,6,...}{\frac{1}{2l+1} \\sum_{m=-l}^l{|a_{l,m}|^n}}
```

Comment:
```suggestion
        AP = \sum_{l=2,4,6,...}{\frac{1}{2l+1} \sum_{m=-l}^l{|a_{l,m}|^n}}
```

Reply:
The math isn't rendering correctly right now so I'd like to fix it before we merge. Maybe this'll address the problem but let's not merge before I have a chance to fix it.